Tests/UIKitTests/UIViewExtensionsTests.swift

```diff
@@ -314,6 +360,22 @@ final class UIViewExtensionsTests: XCTestCase {
         XCTAssertEqual(view1.transform, view3.transform)
     }
 
+    func testHorizontalShake() {
+        let view = UIView()
+        view.horizontalShake(animationType: .easeIn)
+        let currentAnimation = view.layer.animation(forKey: view.layer.animationKeys()?.first ?? "")
+
+        XCTAssertEqual(currentAnimation?.timingFunction, CAMediaTimingFunction(name: .easeIn))
```

Comment:
You don't want to test the fulfilment like the others? Make sure they're back in the original position, maybe?